policyengine_us/variables/gov/local/ca/la/general_relief/eligibility/la_general_relief_eligible.py

```diff
@@ -41,4 +41,5 @@ def formula(spm_unit, period, parameters):
             & net_income_eligible
             & personal_property_eligible
             & immigration_status_eligible
-        ) | disability_eligible
+            & disability_eligible
```

Comment:
this makes people without disabilities ineligible, which doesn't seem right. i think just remove the disability part for now

Reply:
agreed, changes made